### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by yaairfernando**
In this branch we included:

- Form Component
- Add and remove action creator
- List the books
- Set up the books reducer with the add and remove cases

### CODE REVIEW

**Review by fatymahmed — CHANGES_REQUESTED** [review]
## Hi @YairFernando67 @Dandush03,
You have done a great job with this milestone 👏👏, just a few things to change for it to be approved.

**src/index.js**
-  It adds example books to initial state object, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png) from pathwright.

**Book component**
- it is placed in `src/components` directory
- it accepts `Book` object as prop, see  [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Book/AccceptBookObjectAsProp.png) from Pathwright.
- it represents a single row of books with id, title and category  screenshot from Pathwright.

**BooksList component**
- it is placed in `src/containers` directory
- it displays books list in html table using `Book` component, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/BooksAsTable.png) from pathwright
- it passes a `book` object as a prop to each `Book` component

**BookForm component**
-  it is placed in `src/containers` directory
-  it has a state with `title` and `category`

**Redux actions**
- The REMOVE_BOOK action should accept `Book object` as an argument, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/ReduxActions/ReducxActions.png) from pathwright.

### Please make the above changes then request for another review.
## Status: Changes Required ♻️
_Feel free to reach out to me on slack (Fatima) if you have any questions_

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore, compose } from 'redux';
+import App from './components/App';
+import reducers from './reducers';
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+const store = createStore(reducers, composeEnhancers());
+
```
Comment (fatymahmed) [inline src/index.js]: - Add example books here as the initial state

**Comment by Dandush03** [comment]
# Hey! Thanks for the review, Fallowing  @fatymahmed request, we check all feature pointed below 💪 
**src/index.js**
- [x]  It adds example books to initial state object, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png) from pathwright.

**Book component**
- [x] it is placed in `src/components` directory
- [x] it accepts `Book` object as prop, see  [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Book/AccceptBookObjectAsProp.png) from Pathwright.
- [x] it represents a single row of books with id, title and category  screenshot from Pathwright.

**BooksList component**
- [x] it is placed in `src/containers` directory
- [x] it displays books list in html table using `Book` component, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/BooksAsTable.png) from pathwright
- [x] it passes a `book` object as a prop to each `Book` component

**BookForm component**
- [x]  it is placed in `src/containers` directory
- [x]  it has a state with `title` and `category`

**Redux actions**
- [x] The REMOVE_BOOK action should accept `Book object` as an argument, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/ReduxActions/ReducxActions.png) from pathwright.

**Review by fatymahmed — APPROVED** [review]
## Hi @YairFernando67 @Dandush03, 
### You have done a great job in this milestone 🎉🎉
### It's a 👍🏾 from me!
## ✔️ Approved!

**Pull request merged**